File: 4/Problem2.c
#include<stdio.h>

void DecFact(int iNo)
{
    int iCnt = 0;
    for(iCnt = (iNo /2); iCnt >= 1; iCnt--)
    {
        if(iNo % iCnt == 0)
        {
            printf("%d\n",iCnt);
        }
    }

    
}


int main()
{
    int iValue = 0;
    printf("Enter the number :");
    scanf("%d",&iValue);

    DecFact(iValue);
    return 0;
}

File: 20/problem2.java
import java.util.*;

class DigitX
{
    public void SumEven(int iNo)
    {
        int iCnt = 0;
        for(iCnt = iNo ; iCnt >= 1; iCnt--)
        {
            System.out.println(iCnt);
        }        
    }
}
class problem2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        DigitX dobj = new DigitX();

        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        dobj.SumEven(iValue);

    }
}